src/test.jl

```diff
@@ -64,7 +64,6 @@ module TestPaths
     using Dates: Dates
```

Comment:
If we use `git mv src/test.jl ext/FilePathsBaseTestExt.jl` then we could consider renaming this to `test_stub.jl` since that's basically what this file is doing now.